Magento2/Sniffs/Functions/DiscouragedFunctionSniff.php

```diff
@@ -194,24 +194,24 @@ class DiscouragedFunctionSniff extends ForbiddenFunctionsSniff
         '^gettype$' => null,
         '^var_dump$' => null,
         '^tempnam$' => null,
-        '^realpath$' => null,
+        '^realpath$' => '\Magento\Framework\Filesystem\Driver\File->getRealPath()',
         '^linkinfo$' => null,
         '^lstat$' => null,
         '^stat$' => null,
         '^lchgrp$' => null,
         '^lchown$' => null,
         '^show_source$' => null,
-        '^is_dir$' => null,
+        '^is_dir$' => '\Magento\Framework\Filesystem\Driver\File->isDirectory()',
         '^is_executable$' => null,
-        '^is_file$' => null,
+        '^is_file$' => '\Magento\Framework\Filesystem\Driver\File->isFile()',
         '^is_link$' => null,
-        '^is_readable$' => null,
-        '^is_writable$' => null,
-        '^is_writeable$' => null,
+        '^is_readable$' => '\Magento\Framework\Filesystem\Driver\File->isReadable()',
+        '^is_writable$' => 'Magento\Framework\Filesystem\Io\File->isWriteable()',
+        '^is_writeable$' => 'Magento\Framework\Filesystem\Io\File->isWriteable()',
         '^is_uploaded_file$' => null,
         '^glob$' => null,
```

Comment:
This probably could be replaced to `\Magento\Framework\Filesystem\Glob::glob` like there:
https://github.com/magento/magento2/blob/0b968b881498d650aed7b52ed5d78a5833c5da62/lib/internal/Magento/Framework/App/Utility/Files.php#L285